packages/nextra-theme-blog/src/styles.css

```diff
@@ -30,3 +30,7 @@ h1 {
     @apply nx-font-normal;
   }
 }
+
+.nx-prose .nextra-callout p {
+  @apply nx-m-0;
+}
```

Comment:
Preferring this css rule over using `.not-prose` to enable callout content to have markup. Not sure if we want paragraphs inside callouts to have more space around them though. Multiple lines do now look like this:
<img width="680" alt="image" src="https://github.com/shuding/nextra/assets/7533849/04df4377-8846-4f76-ac69-e0631942cbc0">